pkg/virt-handler/options.go

```diff
@@ -71,17 +71,22 @@ func virtualMachineOptions(
 
 func capabilitiesToTopology(capabilities *libvirtxml.Caps) *cmdv1.Topology {
 	topology := &cmdv1.Topology{}
-	if capabilities == nil {
+	if capabilities == nil || capabilities.Host.NUMA == nil {
```

Comment:
My feeling about this patch is that it does not solve the problem. It just adds guards to avoid a segfault.
It wasn't nil before, why it is now? When did this happen?

The guards aren't wrong and the patch might as well avoid a segfault but I would love to know why it became a problem now.

Reply:
This commit looks like the culprit.
https://github.com/kubevirt/kubevirt/commit/5cb6bd3181634cdcba1faa29204bc5a410b55a0b. 

Initial thoughts are there is a difference in how the XML is being unmarshaled with the switch to libvirt. I will try cherry picking this patch to 1.3 since the issue does not appear on that branch and let you know if the error reproduces itself.